src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -69,14 +69,17 @@ public function denormalize(mixed $data, string $type, string $format = null, ar
             }
         }
 
-        if (!\is_int($data) && !\is_string($data)) {
-            throw NotNormalizableValueException::createForUnexpectedDataType('The data is neither an integer nor a string, you should pass an integer or a string that can be parsed as an enumeration case of type '.$type.'.', $data, [Type::BUILTIN_TYPE_INT, Type::BUILTIN_TYPE_STRING], $context['deserialization_path'] ?? null, true);
+        $backingType = (new ReflectionEnum($type))->getBackingType()->getName();
```

Comment:
```suggestion
        $backingType = (new \ReflectionEnum($type))->getBackingType()->getName();
```

and please remove the `use` on L14